x-pack/plugins/synthetics/public/apps/synthetics/components/step_details_page/components/labels.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const LCP_HELP_LABEL = i18n.translate('xpack.synthetics.coreVitals.lcp.help', {
+  defaultMessage:
+    'Largest contentful paint measures loading performance. To provide a good user experience, LCP should occur within 2.5 seconds of when the page first starts loading.',
```

Comment:
Where are these copies from? Looks vague to me. Every metric measures loading performance. We should have this tooltip as something like `LCP measure when the page's main content is loaded.` 

Need to do this for other metrics as well.

Reply:
so for now i just copied what we had in UX app, i will create a follow up issue to get proper definitions from PM and Writer team.